lib_nn/src/asm/vpu_memset_256.S

```diff
@@ -1,6 +1,7 @@
-// Copyright 2020-2021 XMOS LIMITED.
+// Copyright 2020-2025 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
-#ifndef NN_USE_REF
+#if defined(__XS3A__)
```

Comment:
Is this change from NN_USE_REF to XS3A okay?

Reply:
yes as it only runs on xs3a